<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FANItem</class>
 <widget class="QWidget" name="FANItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>60</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>60</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FANItem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px;
background-color: #ffffff;</string>
  </property>
  <widget class="QPushButton" name="m_avatarPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: #ffffff;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="m_nicknameLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>花未眠</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_reasonLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>看到你的博客，感觉非常棒！</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_agreePushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>同意</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_rejectPushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>拒绝</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_statusLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>申请中</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
